### ISSUE TITLE
Address theme overwrites correctly (remove !importants)

### ISSUE THREAD

**Issue by carcruz** [opentargets/issues#3062]
In the UI codebase, we use an exhaust amount of **!important** flags to overwrite mui styles. This is a bad practice and should be removed.

The task is to discover why the mui theme is not overwritten and explore the possibility of migrating styles to the styled-components approach.

**Comment by prashantuniyal02** [comment]
Hi @carcruz and @chinmehta has this been completed?

**Comment by carcruz** [comment]
Yes, in most of the cases, @prashantuniyal02  this can be closed
